controllers/pipeline/cache.go

```diff
@@ -19,7 +19,7 @@ package pipeline
 import (
 	"context"
 	applicationapiv1alpha1 "github.com/redhat-appstudio/application-api/api/v1alpha1"
-	"github.com/redhat-appstudio/integration-service/api/v1alpha1"
+	"github.com/redhat-appstudio/integration-service/api/v1beta1"
```

Comment:
This is not relating to the schema change, but I am thinking maybe we can import this api package in the same way in all files, use `"github.com/redhat-appstudio/integration-service/api/v1beta1"` or `integrationv1beta1 "github.com/redhat-appstudio/integration-service/api/v1beta1"`, then it is easier to copy code in different files during the development or update the schema version the future. Thanks!

Reply:
Good point, I updated all references to use `v1beta1` and removed `integrationv1beta1` named imports.